package.json

```diff
@@ -19,7 +19,7 @@
     "start:bbc-a11y:ci": "npm run build:bbc-a11y && bash -c \"docker run --rm --tty --volume $PWD:/ws bbca11y/bbc-a11y-docker --config /ws/a11y.js --reporter /ws/lib/bbcA11YJUnitReporter.js; exit 0;\" && npm run clean",
     "start:lighthouse:junit": "node runLighthouse.js",
     "start:lighthouse:junit-headless": "A11Y_HEADLESS=true node runLighthouse.js",
-    "start:lighthouse:junit-headless-pretty": "A11Y_HEADLESS=true A11Y_TERSER=true node runLighthouse.js && yarn lighthouse-to-console",
+    "start:lighthouse:headless": "A11Y_HEADLESS=true A11Y_PRETTY=true node runLighthouse.js && npm run lighthouse-to-console",
```

Comment:
I don't think `&& npm run lighthouse-to-console` is needed now that you manually call the console in the code.